### ISSUE TITLE
How about add a light weight framework?

### ISSUE THREAD

**Issue by baiting98** [FuzzyBearBTC/PeercoinBlockExplorer#43]
how about to add a light framework like CodeIgniter ?  It don't need lots of config or require, but we can quick use it to make good url ( like xxx/block/101997),or develop some json api (like http://ppc.blockr.io/api/v1/block/info/1).
We can make a better peercoin explorer site.
if it's sound good ,i will try to setup it.

**Comment by FuzzyBearBTC** [comment]
yup sounds good... :) happy to run the code off a url like API.peercopintalk.org or something to
